packages/js/src/shared-admin/constants/sync.js

```diff
@@ -0,0 +1,31 @@
+export const POST_METADATA_KEYS = {
+	focusKeyphrase: "_yoast_wpseo_focuskw",
+	noIndex: "_yoast_wpseo_meta-robots-noindex",
+	noFollow: "_yoast_wpseo_meta-robots-nofollow",
+	facebookTitle: "_yoast_wpseo_opengraph-title",
+	facebookDescription: "_yoast_wpseo_opengraph-description",
+	facebookImageUrl: "_yoast_wpseo_opengraph-image",
+	facebookImageId: "_yoast_wpseo_opengraph-image-id",
+	twitterTitle: "_yoast_wpseo_twitter-title",
+	twitterDescription: "_yoast_wpseo_twitter-description",
+	twitterImageUrl: "_yoast_wpseo_twitter-image",
+	twitterImageId: "_yoast_wpseo_twitter-image-id",
+	pageType: "_yoast_wpseo_schema_page_type",
+	articleType: "_yoast_wpseo_schema_article_type",
+	isCornerstone: "_yoast_wpseo_is_cornerstone",
+	readabilityScore: "_yoast_wpseo_content_score",
+	seoScore: "_yoast_wpseo_linkdex",
+	inclusiveLanguageScore: "_yoast_wpseo_inclusive_language_score",
+	advanced: "_yoast_wpseo_meta-robots-adv",
+	breadcrumbsTitle: "_yoast_wpseo_bctitle",
+	canonical: "_yoast_wpseo_canonical",
+	wordProofTimestamp: "_yoast_wpseo_wordproof_timestamp",
+	seoTitle: "_yoast_wpseo_title",
+	seoDescription: "_yoast_wpseo_metadesc",
+	readingTime: "_yoast_wpseo_estimated-reading-time-minutes",
+};
+
+export const SYNC_TIME = {
```

Comment:
Interesting point, how do you think it can go wrong? We are subscribing to the store, we listen to the changes of fixed number of values, we don't make api calls, just update the hidden fields or update the `core/editor` store.